src/HealthChecks.SignalR/SignalRHealthCheck.cs

```diff
@@ -32,7 +32,7 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
             }
             finally
             {
-                await connection?.DisposeAsync();
+                 connection?.DisposeAsync().GetAwaiter();
```

Comment:
?

Reply:
In my pr this is written like connection?.DisposeAsync.AsTask();